community-members.md

```diff
@@ -181,6 +181,17 @@ Maintainers:
 - [Francis Bogsanyi](https://github.com/fbogsany), Shopify
 - [Matthew Wear](https://github.com/mwear), NewRelic
 
+## PHP
+
+Repo: [open-telemetry/opentelemetry-php](https://github.com/open-telemetry/opentelemetry-php)
+
+Approvers:
+- [Brain Akins](https://github.com/bakins), Mailchimp
+
+Maintainers:
+- [Bob Strecansky](https://github.com/bobstrecansky), Mailchimp
+- [Dmitry Krokhin](https://github.com/nekufa), basis.company
```

Comment:
@mtwo I think you also talk to someone on a different company to help.

Reply:
@bogdandrutu yes there are some people from Etsy as well, but they have not yet confirmed which role they want.

Reply:
I assume while still in bootstrap phase we can add when they decide.